### ISSUE TITLE
time_sequence_prediction example doesn't work

### ISSUE THREAD

**Issue by Pierre-Bartet** [pytorch/examples#199]
After some iterations, results are totally crazy, I also tried different weight initialization but I didn't succed to make it work. At least it should be marked as broken so that people don't lose time trying to fix their installation because examples are not working.

[predict2.pdf](https://github.com/pytorch/examples/files/1217309/predict2.pdf)
[predict3.pdf](https://github.com/pytorch/examples/files/1217310/predict3.pdf)

**Comment by johnmcaliley** [comment]
@Pierre-Bartet I'm seeing the same results.  Were you able to get this working?  I tried backing out the last commit in the train file to see if it worked, but I still get the same results.

**Comment by Pierre-Bartet** [comment]
Unfortunately no :(

**Comment by sjagter** [comment]
I got it working by using a smaller learning rate of 0.1 (default looks like 1.0).

Line 52: `optimizer = optim.LBFGS(seq.parameters(), lr=0.1)`
[predict14.pdf](https://github.com/pytorch/examples/files/1292293/predict14.pdf)

**Comment by Pierre-Bartet** [comment]
@sjagter Indeed ! I should have tried this instead of switching first to Adam.
